### PR TITLE
this should make the 0020_topic_slug migration reversible

### PR DESCRIPTION
**Pull request by Rowansdabomb**

### CODE REVIEW

File: dispatch/migrations/0021_topic_slug_unique.py [hunk 1]
```diff
@@ -0,0 +1,20 @@
+# -*- coding: utf-8 -*-
```
Comment (psiemens) [inline dispatch/migrations/0021_topic_slug_unique.py]: Was it necessary to create a whole new migration file?

Comment (Rowansdabomb) [reply dispatch/migrations/0021_topic_slug_unique.py]: When applying the reverse_set_defaults function, I ran into a duplicate key error. So the second migration file is just to ensure that before reverting migration 0020, the fields do not have to be unique. Of course as I write this I realize I set the slugs to None, so you are probably right, this 0021 isn't necessary.

**Review by psiemens — APPROVED** [review]

**Comment by psiemens** [comment]
Looks good!

**Comment by jumbosushi** [comment]
Looks awesome :100:

**Pull request merged**